src/__tests__/email-chk.test.ts

```diff
@@ -112,4 +115,14 @@ describe(EmailChk, () => {
             website: 'https://www.dintero.com',
         })).toBe('test@dintero.com');
     });
+
+    it('should use tlds when validating email', () => {
+        const emailChk = EmailChk();
+        expect(emailChk('test@skola.goteborg')).toBe('test@skola.com');
+        expect(emailChk('test@skola.se')).toBe('');
+        expect(emailChk('test@gmail.se')).toBe('test@gmail.com');
+        expect(emailChk('test@dintero.co.uk')).toBe('test@dintero.dk');
+        expect(emailChk('test@netto.finland')).toBe('test@netto.fi');
```

Comment:
this seems strange, we probably should handle TLS that have a seccond-level domain differently as this suggestion does not make sense